packages/snippet-bot/src/violations.ts

```diff
@@ -27,6 +33,79 @@ export interface Violation {
 
 const dataBucket = process.env.DEVREL_SETTINGS_BUCKET || 'devrel-prod-settings';
 
+export const checkRemovingUsedTagViolations = async (
+  changes: ChangesInPullRequest,
+  config: Configuration,
+  parseResults: Map<string, ParseResult>,
+  baseRepositoryPath: string, // googleapis/java-accessapproval
+  baseBranch: string
+): Promise<Map<string, Array<Violation>>> => {
+  const removeUsedTagViolations = new Array<Violation>();
+  const removeConflictingTagViolations = new Array<Violation>();
+  const snippets = await getSnippets(dataBucket as string);
```

Comment:
You shouldn't have to cast this to string - what type does the compiler think it has?

Reply:
It was a leftover because I declared it as `string | undefined`, so now it's not needed.